src/Models/SprintCloseSession.php

```
@@ -0,0 +1,129 @@
+class SprintCloseSession
+{
+    // ===========================
+    // CREATE
+    // ===========================
+
+    /**
+     * Open a new close session for a sprint.
+     *
+     * @return int New session ID
+     */
+    public static function create(Database $db, int $sprintId, int $initiatedBy): int
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)